src/main/java/com/github/mmolimar/kafka/connect/fs/file/reader/EmptyFileReader.java

```diff
@@ -0,0 +1,54 @@
+package com.github.mmolimar.kafka.connect.fs.file.reader;
+
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.kafka.connect.data.Struct;
+
+import java.util.Map;
+
+public class EmptyFileReader extends AbstractFileReader<Void>{
```

Comment:
The test coverage dropped a lot because of this file. It doesn't really make sense to test most of these operations since they will never be called/they do nothing (ie: `seekFile` is never called and is an empty method)